### PR TITLE
GUI tests: Use headless Chrome

### PR DESCRIPTION
**Pull request by ggreer**
This makes it so we don't need to start X, avoiding any issues with multiple xvfbs running on a single worker.

### CODE REVIEW

File: images/tectonic-builder/Dockerfile [hunk 1]
```diff
@@ -54,14 +54,14 @@ RUN cd /tmp && \
     ln -s /usr/local/yarn/bin/yarn /usr/local/bin/yarn
 
 
-### install chrome & xvfb to run the installer-gui tests.
-
+# install chrome for installer gui tests
+# using chome beta because v60 or higher is needed for headless mode
```
Comment (amrutac) [inline images/tectonic-builder/Dockerfile]: nit: <code>chrome<code>

Comment (ggreer) [inline images/tectonic-builder/Dockerfile]: me spel gud

**Comment by ggreer** [comment]
Even in the latest version of Chrome, we still need xvfb to simulate keypresses: https://bugs.chromium.org/p/chromedriver/issues/detail?id=1772

I'll update this PR with some `xvfb-run` stuff, so at least we won't get conflicting displays on the same builder.

**Comment by ggreer** [comment]
Alright. Now this is ready.

**Pull request merged**